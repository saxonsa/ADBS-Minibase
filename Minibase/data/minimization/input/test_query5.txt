Q(x) :- R(x,4), R(x,12), R(u,4), R(y,4),T(u,4)