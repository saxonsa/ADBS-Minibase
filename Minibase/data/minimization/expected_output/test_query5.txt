Q(x, y) :- R(y, x, z), R(x, v, z), T(x, z), R(y, x, 1)
